cspell.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "0.2",
+    "ignorePaths": [],
+    "dictionaryDefinitions": [],
+    "dictionaries": [],
+    "words": [
+        "Catmull",
+        "cornerstonejs",
+        "crosshairs",
+        "kitware",
+        "Labelmap",
+        "segmentations",
+        "toolgroup",
+        "Viewports",
+        "wado"
+    ],
+    "ignoreWords": [],
+    "import": []
+}
```

Comment:
please add these to your user settings and not workspace, so that we don't get these

Reply:
If we are going to consistently use the spelling, then we need a shared dictionary with common words specific to CS3D - would like to reconsider the suggestion to have that not be shared.

Reply:
Since this is extension specific I don't like it, there is no standard spelling check, so I can imaging someone else use another extension and basically add another config file to the repo, since it is different. Ideally you should add these to your user settings on the vscode

![CleanShot 2023-12-13 at 08 30 16](https://github.com/cornerstonejs/cornerstone3D/assets/7490180/c34a0fba-f530-4dff-9216-a8de4848bc3c)